src/storage/file_io.h

```diff
@@ -454,6 +454,10 @@ struct flush_stats
   unsigned int num_tokens;
 };
 
+// *INDENT-OFF*
+using FILEIO_UNLINKED_VOLINFO = std::vector <std::tuple <int, std::string, std::string>>;
```

Comment:
```suggestion
using FILEIO_UNLINKED_VOLINFO_MAP = std::map <int, std::pair< std::string, std::string>>;
```
How about using the dictionary container, `std::map` or `std::unordered_map`, instead? 
in that 
1. It uses a unique key, volid. 
2. The way it is used seems what the dictionary is for so that the code gets simpler and clearer.

Please check the diff on my version with map: https://github.com/wrlawodms/cubrid/commit/52158b9589b77a770082cfb7e628bd7ca3576c0d
13 insertions(+), 34 deletions(-).